logstash-core/lib/logstash-core/logstash-core.rb

```diff
@@ -23,9 +23,8 @@
 # wrapper.
 unless $LS_JARS_LOADED
   jar_path = File.join(File.dirname(File.dirname(__FILE__)), "jars")
-  $:.unshift jar_path
```

Comment:
one less `$LOAD_PATH` entry to search through.
would be best if these were loaded through the `require_jar` but that is smt for a different discussion